rllib/utils/replay_buffers/prioritized_episode_buffer.py

```diff
@@ -128,6 +124,7 @@ def __init__(
         batch_length_T: int = 1,
         alpha: float = 1.0,
         metrics_num_episodes_for_smoothing: int = 100,
+        metrics_num_episodes_for_smoothing: int = 100,
```

Comment:
Arg defined twice?

Reply:
Great catch! Removed in the next commit.